org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/WorkspaceManager.xtend

```diff
@@ -183,7 +183,7 @@ import java.util.HashSet
     }
     
     protected def boolean exists(URI uri) {
-        getProjectManager(uri)?.resourceSet.URIConverter.exists(uri, null)
+        getProjectManager(uri)?.resourceSet?.URIConverter.exists(uri, null)
```

Comment:
No, for XtextResourceSet and ResourceSetImpl it is guaranteed to return not null.